dependency-charts/datapusher/values.yaml

```diff
@@ -6,12 +6,28 @@ replicaCount: 1
 
 image:
   repository: keitaro/ckan-datapusher
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
I think it's better to have `pullPolicy` set to `IfNotPresent`, as before - and define a specific version instead of always pulling `latest`.

Reply:
we use the chart version to lock the image tag. Currently it's 0.0.17

Reply:
I don't think it's smart to use chart version, can we use something like appVersion or similar?

Reply:
I'll add a separate tag field in the values, as it is for the CKAN deployment.